cli/src/cluster_query.rs

```diff
@@ -102,6 +102,19 @@ impl ClusterQuerySubCommands for App<'_, '_> {
                         .index(1),
                 ),
         )
+        .subcommand(
+            SubCommand::with_name("recent-prioritization-fees")
+                .about("Get recent prioritization fees")
+                .arg(
+                    Arg::with_name("account")
+                        .long("account")
+                        .value_name("ACCOUNT")
```

Comment:
```suggestion
                    Arg::with_name("accounts")
                        .long("accounts")
                        .value_name("ACCOUNTS")
```
nit: change arg name to match variable name